tests/testthat/test-util.R

```diff
@@ -34,3 +34,36 @@ test_that("wrap_text() works", {
   expect_equal(wrap_str, "test-modify-model-\nfield.R\ntest-submit-models.R")
   expect_equal(max_line_char(wrap_str), 20)
 })
+
+test_that("wrap_text(): past offenders", {
+  expect_identical(
+    wrap_text("abcde", width = 2, wrap_sym = NULL, strict = TRUE),
+    "ab\ncd\ne"
+  )
+
+  expect_identical(
+    wrap_text("abcdefghij", width = 4, wrap_sym = NULL, strict = TRUE),
+    "abcd\nefgh\nij"
+  )
+
+  expect_identical(
+    wrap_text(
+      "test-foo.R\ntest-foo_bar.R",
+      width = 8, wrap_sym = NULL, strict = TRUE
+    ),
+    "test-foo\n.R\ntest-foo\n_bar.R"
+  )
+
+  expect_identical(
+    wrap_text(
+      "test-foo.R\ntest-foo_bar.R",
+      width = 4, wrap_sym = NULL, strict = TRUE
+    ),
+    "test\n-foo\n.R\ntest\n-foo\n_bar\n.R"
+  )
+
+  expect_identical(
+    wrap_text("foo/bar/baz", width = 4),
+    "foo/\nbar/\nbaz"
```

Comment:
Note that all these cases involve _multiple_ splits on the same separator.  Although I haven't looked into it closely, I think that is why none of these issues where noticed when looking at the bbr example in gh-57.  For example, `man/check_nonmem_table_output.Rd` is split to `man/NEWLINEcheck_nonmem_table_NEWLINEoutput.Rd`.  However, despite that having two newlines inserted, that's one split on "/" and then another on "_".  For any given separator, `paste_pieces()` doesn't get far enough into its logic to hit the bugs.